packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -213,7 +238,18 @@ class ProxyController {
         logCtx: LogContext;
     }) {
         const safeHeaders = proxyService.stripSensitiveHeaders(config.headers, config);
-        await logCtx.info(`${config.method.toUpperCase()} ${url} was successful`, { headers: safeHeaders });
+        await logCtx.http(`${config.method.toUpperCase()} ${url} was successful`, {
+            meta: null,
```

Comment:
meta could potentially be optional. Not a blocker for now though

Reply:
You can definitely removed it from the required field